Mapping/LiSim/sim.py

```diff
@@ -37,25 +37,26 @@ def save_carte(self):
     def give_carte(self):
         return self.carte
 
-    def noise(self, clean_data):
+    def _noise(self, clean_data, dev, esp=0):
         '''Function that adds noise from a normal distribution to the data provided to mimic real world'''
         data = np.array(clean_data)
-        noise = np.random.normal(self.mu,self.std,[int(360/self.angle_step),2])
+        noise = np.random.normal(esp,dev,clean_data.shape)
         return noise+clean_data
 
-    def simulate(self, show=False):
+    def simulate(self, show=False, uPos=2, uDist=2.5, uTheta=1.25):
```

Comment:
I don't understand the use of uPos, Udist and Utheta

Reply:
Differentiating between uncertainties. Exposing them and exposing them separately allows to test more or less accurate LiDARs

Reply:
cool good idea